### PR TITLE
Support StrLen in FT.AGGREGATE

### PR DESCRIPTION
**Pull request by Avital-Fine**
Adds new APLLY function to string functionality - `srtlen`.

MOD-4141

### CODE REVIEW

**Review by ashtul — COMMENTED** [review]
Looks good.
@K-Jo, can we move forward with the `strlen` API?

**Comment by K-Jo** [comment]
LGTM @ashtul @Avital-Fine but the docs are no longer part of the PR?

**Comment by Avital-Fine** [comment]
> LGTM @ashtul @Avital-Fine but the docs are no longer part of the PR?

You're right

File: tests/pytests/test_aggregate.py [hunk 1]
```diff
@@ -713,6 +712,18 @@ def testContains(env):
                                                                  ['t', 'abba'], \
                                                                  ['t', 'abbabb']]))
 
+def testStrLen(env):
+    conn = getConnectionByEnv(env)
+    env.execute_command('ft.create', 'idx', 'SCHEMA', 't', 'TEXT', 'SORTABLE')
+    conn.execute_command('hset', 'doc1', 't', 'aa')
+    conn.execute_command('hset', 'doc2', 't', 'aaa')
+    conn.execute_command('hset', 'doc3', 't', 'ab')
```
Comment (oshadmi) [inline tests/pytests/test_aggregate.py:720]: Consider to also test an empty value `""`

**Review by ashtul — APPROVED** [review]
:100:

**Pull request merged**